sw/device/silicon_creator/lib/cert/asn1.h

```diff
@@ -0,0 +1,200 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef __ASN1__
+#define __ASN1__
+
+#include "sw/device/lib/base/memory.h"
+#include "sw/device/silicon_creator/lib/error.h"
+
+/** Structure holding the state of the asn1 generator.
```

Comment:
nit formatting (applies throughout):

```suggestion
/**
 * Structure holding the state of the asn1 generator.
 * ...
 */
```